### PR TITLE
[bugfix] Unable to specify collector when creating a monitor

### PR DESCRIPTION
**Pull request by Calvin979**
## What's changed?

Link issue: #2300
Test result: 
image

## Checklist

- [x]  I have read the [Contributing Guide](https://hertzbeat.apache.org/docs/community/code_style_and_quality_guide)
- [ ]  I have written the necessary doc or comment.
- [ ]  I have added the necessary unit tests and all cases have passed.

## Add or update API

- [ ] I have added the necessary [e2e tests](https://github.com/apache/hertzbeat/tree/master/e2e) and all cases have passed.

### CODE REVIEW

**Review by tomsun28 — APPROVED** [review]
lgtm👍 close https://github.com/apache/hertzbeat/issues/2300

File: web-app/src/app/routes/monitor/monitor-form/monitor-form.component.html [hunk 1]
```diff
@@ -107,6 +107,7 @@
             [extra]="{collectors}"
             [name]="'collector'"
             [(ngModel)]="collector"
+            (ngModelChange)="onCollectorChange($event)"
```
Comment (kerwin612) [inline web-app/src/app/routes/monitor/monitor-form/monitor-form.component.html:110]: There is no need to add a new event; if necessary, you can get the new value in the submit callback;

Comment (kerwin612) [reply web-app/src/app/routes/monitor/monitor-form/monitor-form.component.html:110]: https://github.com/apache/hertzbeat/blob/91111e55550d7d46a007db2f483b02a2dcd0a1a0/web-app/src/app/routes/monitor/monitor-new/monitor-new.component.ts#L168  
https://github.com/apache/hertzbeat/blob/91111e55550d7d46a007db2f483b02a2dcd0a1a0/web-app/src/app/routes/monitor/monitor-edit/monitor-edit.component.ts#L193  
Changing "`this.collector`" to "`info.collector`" in these two lines seems to be able to meet your requirement.

**Review by kerwin612 — DISMISSED** [review]

File: web-app/src/app/routes/monitor/monitor-form/monitor-form.component.html (hunk 1 above)
Comment (Calvin979) [reply web-app/src/app/routes/monitor/monitor-form/monitor-form.component.html]: Thanks for your suggestion. Please review again when you have time.

**Review by tomsun28 — APPROVED** [review]

**Pull request merged**